subtests/docker_cli/selinux_labels/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Sure, but there's a difference between "let's clean it up some day" and "let's clean it up some day, but in the meantime here's a more maintainable version--already done and tested--that we can accept because we realize that this mythical some-day will never come."